rosserial_vex_cortex/CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Changelog for package rosserial_vex_cortex
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+0.1.1
```

Comment:
I think that all `rosserial` packages should have the same version number so it should be 0.7.7. But I may be wrong. See with @mikepurvis

Reply:
thanks, about to re-commit because my commit broke the building (oops).